src/Locale/en/index.js

```diff
@@ -22,4 +23,5 @@ export default {
   ...Hub,
   ...Notifications,
   ...Shifting,
+  SortOptions,
```

Comment:
voluntarily scoping to prevent name conflicts. (eg: "created_date" sort option conflicts with "created_date": "Created Date")

cc: @khavinshankar